### PR TITLE
Resolves #2003

### LINKED ISSUE #2003: Example2 has Syntax error
There is a - missing before "AllowedAuthenticationType" in example 2 so copying the command doesn't work without correcting for that.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dabf671d-332f-8764-10f3-3741c2095532
* Version Independent ID: adbc1be8-a317-2c53-4612-9df4be686c93
* Content: [Set-VMReplicationServer (hyper-v)](https://docs.microsoft.com/en-us/powershell/module/hyper-v/set-vmreplicationserver?view=win10-ps)
* Content Source: [docset/windows/hyper-v/Set-VMReplicationServer.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/hyper-v/Set-VMReplicationServer.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Fix example 2 typo.
fix: allign formatting in articles. http to https
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2003

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/winserver2012-ps/hyper-v/Set-VMReplicationServer.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Configures a host as a Replica server.
 
 ## SYNTAX
 
-### UNNAMED_PARAMETER_SET_1
+### AuthenticationPort
```
Comment (dariomws) [inline docset/winserver2012-ps/hyper-v/Set-VMReplicationServer.md:17]:
```suggestion
### AuthenticationPort (Default)
```

File: docset/winserver2012r2-ps/hyper-v/Set-VMReplicationServer.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ To display a list of certificates in the computer's My store and the thumbprint
 `PS C:\\\> cd cert:\LocalMachine\My`
 `PS C:\\\> dir | format-list`
 
-For more information about certificate stores, see http://technet.microsoft.com//library/cc757138.aspxhttp://technet.microsoft.com//library/cc757138.aspx.
+For more information about certificate stores, see [Certificate stores](https://technet.microsoft.com//library/cc757138.aspx).
```
Comment (tseward) [inline docset/winserver2012r2-ps/hyper-v/Set-VMReplicationServer.md:155]: This should be changed to:

https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2003/cc757138(v=ws.10)

Comment (e0i) [reply docset/winserver2012r2-ps/hyper-v/Set-VMReplicationServer.md:155]: PR ready when the suggestions have been applied.

cc @velkovb

**Pull request merged**